src/api/goods.js

```diff
@@ -6,6 +6,8 @@ export function getAll() {
     .then(response => response.json());
 }
 
-export const get5First = () => {};
+export const get5First = () => fetch(API_URL)
+  .then(response => response.json());
```

Comment:
DRY. You have three completely same functions here.

`get5First` should return 5 goods already sorted.
Same goes for `getRedGoods`